typescript/sdk/src/ism/schemas.ts

```diff
@@ -10,6 +10,15 @@ const ValidatorInfoSchema = z.object({
   weight: z.number(),
 });
 
+const BaseIsmConfigSchema = z.object({
+  // All the Ism types have an address but in some parts of the code
+  // this value is not set because of the context (ex. warp route config before deployment).
+  // When parsing an object zod's default behavior is to strip unknown fields meaning that
+  // in some cases this field even if it was in the raw object was not included in the parsed value.
+  // (ex. reading a warp route config after using warp read)
```

Comment:
I added it because otherwise it will be reported as a mismatch in the final output, i think that adding the field as optional shouldn't be too much of a problem probably, but happy to check if there might be any undesired side effects